__tests__/hooks/useWalletAddress.test.js

```diff
@@ -17,7 +20,7 @@ const providerWalletAddress = provider.address;
 describe('useWalletAddress()', () => {
   it('returns default type wallet address if no provided type', async () => {
     const wrapper = ({ children }) => (
-      <EtherspotTransactionKit provider={provider}>
+      <EtherspotTransactionKit provider={provider} modular={false}>
```

Comment:
We'll need to remove the modular prop here, it will need to be `accountTemplate="etherspotModular"`

Reply:
removed